packages/harmony/src/components/button/FilterButton/FilterButton.tsx

```diff
@@ -201,22 +196,22 @@ export const FilterButton = forwardRef<HTMLButtonElement, FilterButtonProps>(
                 alignItems='flex-start'
                 justifyContent='center'
                 role='listbox'
-                aria-label={selection?.label ?? label ?? props['aria-label']}
-                aria-activedescendant={selection?.label}
+                aria-label={selectedLabel ?? label ?? props['aria-label']}
+                aria-activedescendant={selectedLabel}
               >
                 {options.map((option) => (
                   <BaseButton
-                    key={option.label}
+                    key={option.value}
                     iconLeft={option.icon}
                     styles={{
                       button: optionCss,
                       icon: optionIconCss
                     }}
                     onClick={() => handleOptionSelect(option)}
-                    aria-label={option.label}
+                    aria-label={option.label ?? option.value}
```

Comment:
Can we do this normalization once above and then replace usages of `option.label` with that computed value?

Reply:
Personally I'd rather keep the map call a simple implied return vs having a fn body with a returned component. This feels easy enough to grok for me to leave as is, but let me know if you feel strongly

Reply:
Nope, don't feel strongly. Just worried if we refactor we might miss one of these values.